.github/workflows/ci.yaml

```diff
@@ -197,9 +197,12 @@ jobs:
         with:
           fetch-depth: 0
 
-      - uses: actions/download-artifact@v3
+      - name: Download artifact
+        uses: dawidd6/action-download-artifact@v2
         id: download
         with:
+          branch: release
+          workflow_conclusion: completed
```

Comment:
Uh whoops...this was added to the wrong workflow. This is supposed to be on the `npm-brew.yaml` workflow